spec/preservation/create_moab_spec.rb

```diff
@@ -0,0 +1,24 @@
+describe Preservation::CreateMoab do
+  context '#create_moab' do
+    let(:mock_so) { instance_double(Moab::StorageObject) }
+    let(:mock_new_version) { instance_double(Moab::StorageObjectVersion) }
+    let(:verification_result) { instance_double(Moab::VerificationResult) }
+
+    it 'returns true if version manifests match the actual files' do
+      expect(mock_so).to receive(:ingest_bag).and_return(mock_new_version)
+      allow(verification_result).to receive(:verified).and_return(true)
+      expect(mock_new_version).to receive(:verify_version_storage).and_return(verification_result)
+      described_class.new.create_moab(mock_so)
+    end
+
+    it 'returns false if version manifests do not match actual files' do
```

Comment:
and this description might be more like "raises ItemError when new version doesn't pass verification"